src/create-verify-archive.js

```diff
@@ -76,6 +77,49 @@ exports.createCommand = function*(argv) {
     var absOutDir = path.resolve(outDir);
 
     yield repoutil.forEachRepo(repos, function*(repo) {
+        if(isWin) {
```

Comment:
We only have evidence of a problem with committers using Windows machine. In theory, this could happen on a Linux/mac machine - but the defaults ensure that this does not happen. Unless this is trivial - I would like us to move forward with getting this fix in for Windows.